pom.xml

```diff
@@ -18,6 +18,7 @@
         <spring.version>4.3.6.RELEASE</spring.version>
         <aspectj.version>1.8.6</aspectj.version>
         <SONAR_LOGIN>Pass in with -D</SONAR_LOGIN>
+        <gnss.artifactory.repo>gnss-artifactory</gnss.artifactory.repo>
```

Comment:
The naming convention for maven properties actually uses a mix of dots and camel case. Camal case is used to separate words, and dots are used to indicate hierarchy.

https://maven.apache.org/pom.html#Properties

gnssArtifactoryRepo

Having said that, you probably don't even need a variable for this. You can just use "s3://gnss-artifactorty/maven-snapots" inline.

Reply:
Either way, it's an artifactory, not a repository.

Reply:
OK, I removed the property and used "s3://gnss-artifactorty/maven-snapshots" etc.

Reply:
:+1: